Rabbitmq queues are now persistent

RPrudon opened the pull request:
Hostname is necessary to make the queues persistent. Rabbitmq generates a random hash hostname on each boot otherwise.

rutgerrademaker commented:
@RPrudon thanks! 
Could you please also update the changelog?

https://github.com/mediact/docker-compose-development-manager/blob/master/CHANGELOG.md

Please also add some more reviewers

RPrudon commented:
@rutgerrademaker I updated the pull request with a changelog and also fixed a typo. I'm unable to add reviewers as I do not have write access to this repo.

rutgerrademaker reviewed (commented):
Looks good!
